language-design.md

```diff
@@ -17,6 +17,97 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+*“According to findings by researchers from Southern Illinois University, this 
+reaction isn’t just because you’re a n00b: they found that Perl, a major programming 
+language used by untold zillions of developers, is no more intuitive to novices than a 
+language with a randomly generated syntax. Why shouldn’t those "interfaces" be humanely 
+designed? … They created a "placebo language" called Randomo, whose syntax was randomly 
+generated, to use in trials alongside Quorum and Perl. Novice programmers were able to write 
+sample programs more accurately in Quorum versus Perl — an interesting, but not terribly 
+surprising, result. More surprising was how Perl compared to Randomo. To quote the paper: 
+'Perl users were unable to write programs more accurately than those using a language designed 
+by chance.' … Every piece of software we use was written by other people, slaving away over 
+thousands of lines of code. Why shouldn’t those "interfaces" be as humanely designed as 
+the ones we tap and swipe?”* 
+
+This quote really resonated with me because my first experience with “programming” was with 
```

Comment:
This is really interesting to see such a different perspective! I also had a chance to try out the MIT app inventor during the summer between my first and second years at college.  I had a somewhat less smooth transition to using the MIT app inventor.  I had also just begun programming, my only experience being CS5 and CS60 at Mudd.  Still, this introduction to programming was enough for me to have difficulty learning the expectations and style of the code block programming of MIT App Developer and Scratch (we were studying languages designed to help introduce people to programming without having to contend with too many syntax errors).

I think it is really interesting that my experience with text-based syntax was enough to make it quite difficult for me to get into the paradigm embodied by block based programming languages.  I remember struggling at length when attempting to perform dynamic object creation in MIT's app inventor and wishing I could go back to using python and the like.